contrib/experimental/great_expectations_experimental/expectations/expect_column_values_reverse_geocoded_lat_lon_to_contain.py

```diff
@@ -0,0 +1,240 @@
+    success_keys = (
+        "mostly",
+        "word",
+        "provider",
+    )
+
+    # This dictionary contains default values for any parameters that should have default values
+    default_kwarg_values = {
+        "provider": None,
+        "mostly": 1,
+    }
+
+    # This object contains metadata for display in the public Gallery
+    library_metadata = {
+        "tags": [
+            "geospatial",
+            "hackathon-2022",
+        ],  # Tags for this Expectation in the Gallery
+        "contributors": [  # Github handles for all contributors to this Expectation.
+            "@mmi333",  # Don't forget to add your github handle here!
+        ],
+        "requirements": ["geopandas"],
+    }
+
+    @classmethod
+    @renderer(renderer_type="renderer.prescriptive")
+    @render_evaluation_parameter_string
+    def _prescriptive_renderer(
+        cls,
+        configuration: ExpectationConfiguration = None,
+        result: ExpectationValidationResult = None,
+        language: str = None,
+        runtime_configuration: dict = None,
+        **kwargs,
+    ) -> List[
+        Union[
+            dict,
+            str,
+            RenderedStringTemplateContent,
+            RenderedTableContent,
+            RenderedBulletListContent,
+            RenderedGraphContent,
+            Any,
+        ]
+    ]:
+        runtime_configuration = runtime_configuration or {}
+        include_column_name = runtime_configuration.get("include_column_name", True)
+        include_column_name = (
+            include_column_name if include_column_name is not None else True
+        )
+        styling = runtime_configuration.get("styling")
+        params = substitute_none_for_missing(
+            configuration.kwargs,
+            [
+                "column",
+                "word",
+                "provider",
+                "mostly",
+            ],
+        )
+
+        if params["mostly"] is None:
+            template_str = "values must be lat lon and contain $word when reverse geocoded by $provider"
+        else:
+            if params["mostly"] is not None:
+                params["mostly_pct"] = num_to_str(
+                    params["mostly"] * 100, precision=15, no_scientific=True
+                )
+                template_str += ", at least $mostly_pct % of the time."
+            else:
+                template_str += "."
+
+        if include_column_name:
+            template_str = f"$column {template_str}"
+
+        return [
+            RenderedStringTemplateContent(
+                **{
+                    "content_block_type": "string_template",
+                    "string_template": {
+                        "template": template_str,
+                        "params": params,
+                        "styling": styling,
+                    },
+                }
+            )
+        ]
+
+
+if __name__ == "__main__":
+    ExpectColumnValuesReverseGeocodedLatLonToContain().print_diagnostic_checklist()
```

Comment:
@mmi333 It looks like we're hitting a linting error on CI/CD here; just need a newline and we're good to merge!

Reply:
@austiezr Should be fixed now!